server/db/helper.js

```diff
@@ -114,6 +114,25 @@ const getUserBadges = (userId) => UserBadge.findAll({
 });
 
 
+const getCompletedCourse = (userId) => UserBadge.findAll({
+  where: {
+    idUser: userId
+  },
+
+  attributes: ['idBadge']
+})
+.then((courses) => {
```

Comment:
What you are retrieving from the userBadge table isn't the course or course Id but the badge's id, can you rename this variable.